required/latex-lab/latex-lab-text.dtx

```diff
@@ -16,11 +16,16 @@
 %
 % for those people who are interested or want to report an issue.
 %
-\def\ltlabtextdate{2024-09-18}
-\def\ltlabtextversion{0.85b}
+\def\ltlabtextdate{2025-02-18}
+\def\ltlabtextversion{0.85e}
 
 %<*driver>
 \documentclass{l3doc}
+% for the phoneme argument
+\DeclareUnicodeCharacter{02C8}{\texttt{\textquotesingle}}
+\DeclareUnicodeCharacter{02D0}{\texttt{:}}
+\DeclareUnicodeCharacter{025B}{$\epsilon$}
```

Comment:
not sure it is ever important but shouldn't that use \m@th?

Reply:
No idea, I simply wanted something that compiles at all and more or less looks right. Something using tipa would probably be better - or we enable compilation with lualatex ;-)